vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/dao/mysql/EngineData.java

```diff
@@ -79,7 +79,7 @@ public void add(Engine engine) throws DAOException {
             ps.close();
 
         } catch (SQLException ex) {
-            if (ex.getMessage().contains("Duplicate entry")) {
+            if (ex.getMessage().contains("Unique index or primary key violation")) {
```

Comment:
As in ClassData, we need to verify the error message.
Maybe add an OR to consider both error message (the old and the new).